sda/cmd/s3inbox/healthchecks_test.go

```diff
@@ -81,7 +81,7 @@ func (suite *HealthcheckTestSuite) TestHealthchecks() {
 	conf, err := config.NewConfig("s3inbox")
 	assert.NoError(suite.T(), err)
 	assert.NotNil(suite.T(), conf)
-	h := NewHealthCheck(8888,
+	h := NewHealthCheck(5555,
```

Comment:
I don't think it's a problem to have the same port number since you close it every time.